Makefile

```diff
@@ -25,6 +25,8 @@ clean:
 
 compile_cairo:
 	cairo-compile cairo_syscalls/syscalls.cairo --output cairo_syscalls/syscalls.json && \
+	cairo-compile cairo_programs/fibonacci.cairo --output cairo_programs/fibonacci.json && \
```

Comment:
i think this should use starknet-compile instead of cairo-compile to execute properly

Reply:
yes, i have already fixed